modules/azurerm/Azure-Container-Registry-Token/token.tf

```diff
@@ -0,0 +1,40 @@
+# -------------------------------------------------------------------------------------
+#
+# Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# --------------------------------------------------------------------------------------
+
+resource "azurerm_container_registry_scope_map" "registry_scope_map" {
+  name                    = "${var.scope_map_name}-scopemap"
```

Comment:
Follow the same naming convention as 
```
name   = join("-", ["scpm", var.scope_map_name])
```

Change `scpm` for a better value if it is not appropriate